README.md

```diff
@@ -152,10 +152,18 @@ metadata:
   name: fenceagentsremediationtemplate-default
   namespace: default
 spec:
-  template: {}
+  template:
+    spec:
+        remediationStrategy: <remediation_strategy>
 ```
 
 > *Note*: FenceAgentsRemediationTemplate CR must be created in the same namespace that the FAR operator has been installed.
+ 
+The `.spec.remediation_strategy` field defines the _remediationStrategy_ that FAR supports and will be used for recovering workloads faster during node reboot in a [non-graceful](https://kubernetes.io/docs/concepts/cluster-administration/node-shutdown/#non-graceful-node-shutdown) manner. There are only two possible remediations - `ResourceDeletion` or `OutOfServiceTaint`:
+
+- `ResourceDeletion`: This remediation strategy (forcefully) removes the pods on the node.
+- `OutOfServiceTaint`: This remediation strategy implicitly causes the removal of the pods and associated volume attachments on the node, rather than the removal of the node object. It achieves this by placing the `OutOfServiceTaint` taint on the node. The `OutOfServiceTaint` strategy also represents a non-graceful node shutdown. A non-graceful node shutdown occurs when a node is shutdown and not detected, instead of triggering an in-operating system shutdown.
```

Comment:
>The `OutOfServiceTaint` strategy also represents a non-graceful node shutdown. A non-graceful node shutdown occurs when a node is shutdown and not detected, instead of triggering an in-operating system shutdown.

I am not sure the above sentence is needed as it was already mentioned in line 162.


```suggestion
- `OutOfServiceTaint`: This remediation strategy implicitly causes the removal of the pods and associated volume attachments on the node. It achieves this by placing the [`OutOfServiceTaint` taint](https://kubernetes.io/docs/reference/labels-annotations-taints/#node-kubernetes-io-out-of-service) on the node.
```